src/web/assets/cp/dist/js/Craft.js

```diff
@@ -12596,17 +12596,35 @@ Craft.EditableTable = Garnish.Base.extend(
         $tbody: null,
         $addRowBtn: null,
 
+        maxRows: null,
+        minRows: null,
+        rowCount: 0,
+        hasMaxRows: false,
+        hasMinRows: false,
+
         radioCheckboxes: null,
 
-        init: function(id, baseName, columns, settings) {
+        init: function(id, baseName, columns, settings, maxRows, minRows) {
+
             this.id = id;
             this.baseName = baseName;
             this.columns = columns;
             this.setSettings(settings, Craft.EditableTable.defaults);
             this.radioCheckboxes = {};
+            this.maxRows = maxRows;
+            this.minRows = minRows;
+
+            if (this.maxRows !== null) {
+                this.hasMaxRows = true;
+            }
+
+            if (this.minRows !== null) {
+                this.hasMinRows = true;
+            }
```

Comment:
For clarity, set `hasMinRows` and `hasMaxRows` set to null up at the top of the file, and try this instead here:

```js
this.hasMinRows = this.minRows !== null;
this.hasMaxRows = this.maxRows !== null;
```